packages/fxa-payments-server/src/lib/validator.test.tsx

```diff
@@ -1,146 +1,171 @@
-import React, { useState, useCallback, useContext } from 'react';
+import React, { useCallback, useContext } from 'react';
 import { render, cleanup, fireEvent } from '@testing-library/react';
 import 'jest-dom/extend-expect';
 import {
+  mainReducer,
   useValidatorState,
   Validator,
   MiddlewareReducer as ValidatorMiddlewareReducer,
   State as ValidatorState,
   defaultState as validatorDefaultState,
+  Action as ValidatorAction,
 } from './validator';
 
 afterEach(cleanup);
 
-it('supports registering a field', () => {
-  const { state } = runAgainstValidator(
-    v => v.registerField({ name: 'foo', fieldType: 'input', required: true }),
-    v =>
-      v.registerField({
-        name: 'bar',
+describe('mainReducer', () => {
+  it('returns the original state when given an unknown action', () => {
+    const state = validatorDefaultState;
+    const action = { type: 'bogus' };
+    const newState = mainReducer(state, action as ValidatorAction);
+    expect(newState).toEqual(state);
+  })
+});
+
+describe('useValidatorState', () => {
+  it('does not throw error when missing initial state or middleware', () => {
+    const Subject = () => {
+      const validator = useValidatorState();
+      return (
+        <div>{JSON.stringify(validator.state)}</div>
+      )
+    };
+    render(<Subject />);
+  })
+});
+
+describe('Validator', () => {
```

Comment:
Most of the changes in this file are just me having re-indented the existing tests under a new `describe()` call.